### PR TITLE
Adding mode and 95th percentile to InsertSizeMetrics.

### PR DESCRIPTION
**Pull request by nh13**
I find that I parse the histogram often for these metrics, and so I figured I should just go ahead and add them officially.

### CODE REVIEW

**Comment by nh13** [comment]
@yfarjoun or @lbergelson bump?

File: src/main/java/picard/analysis/directed/InsertSizeMetricsCollector.java [hunk 1]
```diff
@@ -168,6 +169,7 @@ public void addMetricsToFile(final MetricsFile<InsertSizeMetrics,Integer> file)
                             if (percentCovered >= 0.7 && metrics.WIDTH_OF_70_PERCENT == 0) metrics.WIDTH_OF_70_PERCENT = distance;
                             if (percentCovered >= 0.8 && metrics.WIDTH_OF_80_PERCENT == 0) metrics.WIDTH_OF_80_PERCENT = distance;
                             if (percentCovered >= 0.9 && metrics.WIDTH_OF_90_PERCENT == 0) metrics.WIDTH_OF_90_PERCENT = distance;
+                            if (percentCovered >= 0.9 && metrics.WIDTH_OF_95_PERCENT == 0) metrics.WIDTH_OF_95_PERCENT = distance;
```
Comment (ktibbett) [inline src/main/java/picard/analysis/directed/InsertSizeMetricsCollector.java]: @nh13 shouldn't this be  percentCovered >= 0.95 ?

Comment (nh13) [reply src/main/java/picard/analysis/directed/InsertSizeMetricsCollector.java]: <swearing ensues>

**Comment by nh13** [comment]
@ktibbett one more time, omg that mistake.

**Comment by yfarjoun** [comment]
I'm concerned that the tests passed with that error....could we have tests that would fail on that error?

**Comment by nh13** [comment]
@yfarjoun of course, done.

File: src/test/java/picard/analysis/CollectInsertSizeMetricsTest.java [hunk 2]
```diff
@@ -242,4 +260,82 @@ public void testMultipleOrientationsForHistogram() throws IOException {
...
+        Assert.assertEquals(metric.WIDTH_OF_20_PERCENT, 1 + insertBy*2);
+        Assert.assertEquals(metric.WIDTH_OF_30_PERCENT, 1 + insertBy*4);
+        Assert.assertEquals(metric.WIDTH_OF_40_PERCENT, 1 + insertBy*6);
+        Assert.assertEquals(metric.WIDTH_OF_50_PERCENT, 1 + insertBy*8);
+        Assert.assertEquals(metric.WIDTH_OF_60_PERCENT, 1 + insertBy*10);
+        Assert.assertEquals(metric.WIDTH_OF_70_PERCENT, 1 + insertBy*12);
+        Assert.assertEquals(metric.WIDTH_OF_80_PERCENT, 1 + insertBy*14);
+        Assert.assertEquals(metric.WIDTH_OF_90_PERCENT, 1 + insertBy*16);
+        Assert.assertEquals(metric.WIDTH_OF_95_PERCENT, 1 + insertBy*18);
+        Assert.assertEquals(metric.WIDTH_OF_99_PERCENT, 1 + insertBy*20);
```
Comment (nh13) [inline src/test/java/picard/analysis/CollectInsertSizeMetricsTest.java]: Look Ma, different values for each!

Comment (ktibbett) [reply src/test/java/picard/analysis/CollectInsertSizeMetricsTest.java]: 👍

**Comment by ktibbett** [comment]
Thanks @nh13 : 👍

**Pull request merged**